### PR TITLE
Typo in _all_uniprot

### PR DESCRIPTION
**Pull request by dbdimitrov**
Related to https://github.com/biocypher/metalinks/issues/4

### CODE REVIEW

File: pypath/inputs/uniprot.py [hunk 1]
```diff
@@ -343,7 +343,7 @@ def get_uniprot_sec(organism = 9606):
     if _organism:
 
         from pypath.inputs import uniprot_db
-        proteome = uniprot_db.all_uniprots(organism=organism)
+        proteome = uniprot_db._all_uniprots(organism=organism)
```
Comment (deeenes) [inline pypath/inputs/uniprot.py:346]: @dbdimitrov Where have you encountered an error from this? As far as I'm aware, [this function is called](https://github.com/saezlab/pypath/blob/9cab02d68a31e7ddaf64ba9ed2965148991da423/pypath/inputs/uniprot_db.py#L40) `all_uniprots` and `get_uniprot_sec` runs fine for me.

Comment (dbdimitrov) [reply pypath/inputs/uniprot.py:346]: hmmm, honestly not 100% sure, I believe it was related to `map_names`

Comment (dbdimitrov) [reply pypath/inputs/uniprot.py:346]: PS. I just noticed that actually you changed it in a following version, maybe this was resolved:
The issue that I was encountering was in `0.15.13`.

Feel free to close if `map_names` works for you :)

**Comment by npalacioescat** [comment]
Closed since this was already solved

**Pull request closed without merging**